shared/AppInsightsCore/src/JavaScriptSDK.Interfaces/IAppInsightsCore.ts

```diff
@@ -74,6 +74,8 @@ export interface IAppInsightsCore extends IPerfManagerProvider {
 
     pollInternalLogs?(eventName?: string): number;
 
+    stopPollingInternalLogs?(internalLogPoller?: number): void;
```

Comment:
This is still exposing the internal timer...

Lets move the _internalLogPoller to this class so then the stop doesn't take any arguments

Reply:
And as the AppInsightsCore is using dynamicProto we won't need to expose _internalLogPoller on the instance either, it can remain within the closure -- for even better minification